app/components/list-widget/component.js

```diff
@@ -188,7 +188,7 @@ export default Ember.Component.extend({
             }
```

Comment:
Passing an empty object to transitionToFacet is not handled here, so your change in L191 may have syntax error if facet is defined. It looks brittle. 

Also Reuse ```facetdash``` variable for ```this.get('item.facetDash')```